### PR TITLE
Fix removing plugins

### PR DESCRIPTION
**Pull request by cpuguy83**

### CODE REVIEW

**Comment by cpuguy83** [comment]
ping @anusha-ragunathan @tiborvass

**Comment by cpuguy83** [comment]
ah damn, extra code got in there, one sec.

**Comment by cpuguy83** [comment]
All 💚

File: plugin/manager.go [hunk 1]
```diff
@@ -58,7 +58,12 @@ func (p *plugin) Client() *plugins.Client {
 }
 
 func (p *plugin) Name() string {
-	return p.p.Name
+	name := p.p.Name
+	if len(p.p.Tag) > 0 {
+		// TODO: this feels hacky, maybe we should be storing the distribution reference rather than splitting these
```
Comment (tiborvass) [inline plugin/manager.go]: i'll have to think about this

**Comment by tiborvass** [comment]
LGTM

**Comment by thaJeztah** [comment]
LGTM, thanks!

**Comment by thaJeztah** [comment]
These changes were included in https://github.com/docker/docker/issues/23580, so this was closed in favor of https://github.com/docker/docker/issues/23580

**Pull request closed without merging**